ui/app/styles/core/buttons.scss

```diff
@@ -280,22 +280,6 @@
   margin: 0 !important;
 }
 
-.button.next-feature-step {
```

Comment:
Thanks for deleting this! I realize we'll delete this whole class at the end of the release, but it's nice to remove this one-offs as we go!